ProcessMaker/processmaker
# FOUR-13156: It is to possible save the configuration on the process with description empty from LaunchPad.

## How to Test
-  Open Any Process with "Open in Launchpad"
- Clear description field
- Try to save (Modal window won`t close until description is entered)
- Close modal window
- Open again Edit in Launchpad, previous Descrition is shown

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-13156

## Code Review Checklist
ci:deploy
ci:next
